plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -38,6 +38,8 @@ def execute!(*_commands)
     def wait_for_deploys_to_finish(release)
       start = Time.now
       stable_ticks = CHECK_STABLE / TICK
+      expected = release.release_docs.to_a.sum(&:desired_pod_count)
```

Comment:
I tried this, but it seems the desired_pod_count doesn't exist. At least in my version (I tried a small back port, maybe it was that)